misc/integrated_game_data.sql

```diff
@@ -3,15 +3,15 @@ INSERT INTO users
     (first_name, last_name, email, birth_date, role, grade_level, language_id, high_school_id)
 VALUES
     ('admin', 'name', 'a@email.com', '01/01/2000'::date, 'super-admin', 1, 1, 1),
-    ('macedonian 1', 'name', 'b@email.com', '01/01/2001'::date, 'student', 2, 1, 1),
+    ('macedonian 1', 'name', 'b@email.com', '01/01/2001'::date, 'student', 2, 2, 1),
```

Comment:
Fixed language assignments.

Reply:
How to use this code? Are you wiping the database? Otherwise, new users will have new ideas that make your ensuing code irrelevant--since it relies on knowing ids.

More generally, do you have a database filling script that starts with constants like high school and languages etc.

Reply:
The database fills for the languages and high school additions are in the database migrations, so they'll be up whenever the app spins up.

These are just used as dummy data for development. I just manually run these on my DB IDE. They shouldn't be used in production.

Reply:
Great. Now that the make / alembic thing is fixed, I'm getting the constants into the database after `make clean_all` `make upgrade_db`!

Reply:
If you do `make clean_all`, you should just have to run `make` and it'll do the build, docker container startup, and database building for you. Those three commands are bundled into the default make command.